modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1030,6 +1030,40 @@ m|WARNING
 m|GENERIC
 |===
 
+[#_neo_clientnotification_statement_unsatisfiablerelationshiptypeexpression]
+=== Neo.ClientNotification.Statement.UnsatisfiableRelationshipTypeExpression
```

Comment:
```suggestion
=== Neo.ClientNotification.Statement.UnsatisfiableRelationshipTypeExpression  (when run on version 5.4 or newer)
```

Since all notifications under this category was added on 5.4, I think we can put it up here instead. The question is what we should do if we in the future add another notification with same status code? Maybe we should put `(when run on version x.y or newer)` on the title and also on examples which were added on a different version?

Then we should update`Neo.ClientNotification.Statement.RepeatedRelationshipReference` to be

```
=== Neo.ClientNotification.Statement.RepeatedRelationshipReference (when run on version 5.5 or newer)
...
.Binding a relationship variable more than once 
...
.Binding a variable-length relationship variable more than once (when run on version 5.6 or newer)

```